functions.php

```diff
@@ -20,10 +24,22 @@ function elementor_hello_theme_setup() {
 
 	add_theme_support( 'woocommerce' );
 
+	register_nav_menus(
+		array( 'main-menu' => __( 'Main Menu', 'elementor-hello-theme' ) )
+	);
+
 	load_theme_textdomain( 'elementor-hello-theme', get_template_directory() . '/languages' );
 }
 add_action( 'after_setup_theme', 'elementor_hello_theme_setup' );
 
+// add Threaded comments script
```

Comment:
It's already enqueued at https://github.com/pojome/elementor-hello-theme/blob/master/comments.php#L7